pkg/operator/operator.go

```diff
@@ -147,6 +147,7 @@ func New(config operatorconfig.Config, kubeConfig *rest.Config) (*Operator, erro
 				operatorcontroller.DefaultOperandNamespace:               {},
 				operatorcontroller.DefaultCanaryNamespace:                {},
 				operatorcontroller.GlobalMachineSpecifiedConfigNamespace: {},
+				"openshift-operators":                                    {},
```

Comment:
Please make a global constant in the operatorcontroller package for "openshift-operators" namespace.  Just in case it changes, we can change it in one place.